### PR TITLE
ME18: Removed USB DMA registers and unsupported WUT modes. 

### PR DESCRIPTION
**Pull request by sihyung-maxim**
Changes:
- Removed PWM mode from WUT.
- Removed USB DMA registers.
- Resolved SVDConv warnings.

### CODE REVIEW

File: Libraries/CMSIS/Device/Maxim/MAX32690/Include/hpb_regs.h [hunk 1]
```diff
@@ -176,8 +176,8 @@ typedef struct {
  * @ingroup  hpb_registers
  * @defgroup HPB_MEMBADDR HPB_MEMBADDR
  * @brief    Memory Base Address register. This is the base address of the addressable memory
- *           region in this microcontroller’s RAM. Because the addressable memory is mapped
- *           in 16M boundaries, the lower 24 bits are fixed to 0.
+ *           region in this microcontrollers RAM. Because the addressable memory is mapped in
```
Comment (lorne-maxim) [inline Libraries/CMSIS/Device/Maxim/MAX32690/Include/hpb_regs.h:179]: 's was correct here.

File: Libraries/CMSIS/Device/Maxim/MAX32690/Include/max32690.svd [hunk 2]
```diff
@@ -5548,7 +5548,7 @@ least 1 entry in RX FIFO during this time, this bit is clear by write 1</descrip
      <dim>2</dim>
      <dimIncrement>4</dimIncrement>
      <name>MEMBADDR[%s]</name>
-     <description>Memory Base Address register. This is the base address of the addressable memory region in this microcontroller’s RAM. Because the addressable memory is mapped in 16M boundaries, the lower 24 bits are fixed to 0.</description>
+     <description>Memory Base Address register. This is the base address of the addressable memory region in this microcontrollers RAM. Because the addressable memory is mapped in 16M boundaries, the lower 24 bits are fixed to 0.</description>
```
Comment (lorne-maxim) [inline Libraries/CMSIS/Device/Maxim/MAX32690/Include/max32690.svd:5551]: 's was correct.

File: Libraries/PeriphDrivers/Source/HPB/hpb_reva.svd [hunk 3]
```diff
@@ -239,7 +239,7 @@
         <dim>2</dim>
         <dimIncrement>4</dimIncrement>
         <name>MEMBADDR[%s]</name>
-        <description>Memory Base Address register. This is the base address of the addressable memory region in this microcontroller’s RAM. Because the addressable memory is mapped in 16M boundaries, the lower 24 bits are fixed to 0.</description>
+        <description>Memory Base Address register. This is the base address of the addressable memory region in this microcontrollers RAM. Because the addressable memory is mapped in 16M boundaries, the lower 24 bits are fixed to 0.</description>
```
Comment (lorne-maxim) [inline Libraries/PeriphDrivers/Source/HPB/hpb_reva.svd:242]: 's was correct

**Review by lorne-maxim — CHANGES_REQUESTED** [review]
Did I miss where the USB DMA references were removed in these files?  I didn't see the same type of changes that PR#204 had.

File: Libraries/CMSIS/Device/Maxim/MAX32690/Include/max32690.svd (hunk 2 above)
Comment (sihyung-maxim) [reply Libraries/CMSIS/Device/Maxim/MAX32690/Include/max32690.svd]: Had to revise this description. The ' was not supported in SVDConv.

File: Libraries/CMSIS/Device/Maxim/MAX32690/Include/hpb_regs.h (hunk 1 above)
Comment (sihyung-maxim) [reply Libraries/CMSIS/Device/Maxim/MAX32690/Include/hpb_regs.h]: Had to revise this description. The ' was not supported in SVDConv.

**Comment by sihyung-maxim** [comment]
> Did I miss where the USB DMA references were removed in these files? I didn't see the same type of changes that PR#204 had.

Nope, I forgot to remove them for the ME18.

File: Libraries/CMSIS/Device/Maxim/MAX32690/Include/hpb_regs.h [hunk 4]
```diff
@@ -229,7 +227,7 @@ typedef struct {
 /**
  * @ingroup  hpb_registers
  * @defgroup HPB_MEMTIM HPB_MEMTIM
- * @brief    Memory timing register.
+ * @brief    Hyperbus emory Timing Register.
  * @{
```
Comment (lorne-maxim) [inline Libraries/CMSIS/Device/Maxim/MAX32690/Include/hpb_regs.h:231]: Missing an 'M'

File: Libraries/CMSIS/Device/Maxim/MAX32690/Include/max32690.svd [hunk 5]
```diff
@@ -5701,7 +5701,7 @@ memory.
      <dim>2</dim>
      <dimIncrement>4</dimIncrement>
      <name>MEMTIM[%s]</name>
-     <description>Memory timing register.</description>
+     <description>Hyperbus emory Timing Register.</description>
```
Comment (lorne-maxim) [inline Libraries/CMSIS/Device/Maxim/MAX32690/Include/max32690.svd:5704]: Missing 'M'

File: Libraries/PeriphDrivers/Source/HPB/hpb_reva.svd [hunk 6]
```diff
@@ -392,7 +392,7 @@ memory.
         <dim>2</dim>
         <dimIncrement>4</dimIncrement>
         <name>MEMTIM[%s]</name>
-        <description>Memory timing register.</description>
+        <description>Hyperbus emory Timing Register.</description>
```
Comment (lorne-maxim) [inline Libraries/PeriphDrivers/Source/HPB/hpb_reva.svd:395]: Missing 'M'

**Review by lorne-maxim — APPROVED** [review]
A couple of cosmetic changes.  The rest looks good.

File: Libraries/CMSIS/Device/Maxim/MAX32690/Include/hpb_regs.h (hunk 4 above)
Comment (sihyung-maxim) [reply Libraries/CMSIS/Device/Maxim/MAX32690/Include/hpb_regs.h:231]: Whoops, nice catch. I resolved the typo.

**Pull request merged**